source/quick-start/index.rst

```diff
@@ -131,7 +131,7 @@ it onto your order confirmation page just before the closing ``</body>`` tag::
 For production environments you can send your spec sheets directly to us. We
 will take care of inserting the correct measurements into our database. Feel
 free to get in touch with your sales contact or our `sales team
-<http://www.virtusize.com/contact>`__ for more details on this.
+<sales@virtusize.com>`__ for more details on this.
```

Comment:
I'm lost on this trailing underscore syntax. What's it for? And why do some links end with one underscore and others (like here) with two?
Is it maybe used as in markdown, to enclose something that is supposed to be emphasized? (in which case this is broken, since nothing is enclosed)